### DISCUSSION TITLE
Difference between Box and Paper

### DISCUSSION THREAD

**Discussion by thr3a** [mantinedev/mantine#4877] [Q&A] [answered]
What are the differences between box component and Paper component?
I don't know how to use the two components.

**Comment by mattkrins** [comment, marked as answer]
[Box](https://mantine.dev/core/box/) is the base of all other Mantine components, it's like the equivalent of a simple html div, but with the Mantine props for styling etc.
[Paper](https://mantine.dev/core/paper/) extends the Box component by having a light/dark background based on the color theme.

**Reply by thr3a** [reply]
Thanks!
